exploration/exact-match-selector-options.md

```diff
@@ -0,0 +1,166 @@
+and issues raised in various design documents, notably
+<a href="https://github.com/unicode-org/message-format-wg/pull/471">#471</a>
+(about number selection).
+
+The exact match selector function is an "all-purpose" built-in selector
+for use in messages when a more specific selector (such as for numbers or
+dates) is not available.
+Other selectors, such as `:number` or `:datetime`
+that need to match specific values provide similar functionality.
+Custom selectors should model their exact-match on this selector.
+
+## Use-Cases
+
+_What use-cases do we see? Ideally, quote concrete examples._
+
+As a developer, I would like to create matches based on arbitrary enumerated string
+values in a manner similar to ICU MF1's `SelectFormat`.
+This allows me to use an ordinary enum or data values already present in my data
+to do pattern selection.
+
+As a user, I would like to create numeric matches based on a specific value
+not just the plural or ordinal category of the value.
+This is for cases where the pattern selected is tied to the actual value
+rather than just to the grammatical needs of the language.
+Consider the different between the messages in:
+>```
+>.match {$numChances :number}
+>1   {{You have one chance left}}
+>one {{You have {$numChances}} chance left}}
+>*   {{You have {$numChances}} chances left}}
+>```
+
+As an implementer, I want to create value matches that are not strictly
+tied to the string serialization for data types that I know.
+> For example, if a date value were serialized as
+> `2023-12-08T12:00:00Z[America/Los_Angeles]` and the selector were
+> `{$myDate :date dateStyle=long}` **_and_** I want to do select on the 
+> date value, I might want a match like this:
+>```
+>.match {$myDate}
+>2023-12-08 {{This only fires on this specific date}}
+>* {{This fires some other time}}
+>```
+>A different example that makes this clearer might be numbers.
+> In the `de-DE` locale, the number `42.3` is usually formatted
+> as `42,3`. Consider the message:
+> ```
+> .local $myNum = {|42.3| :number minFractionDigits=1}
+> .match {$myNum :equals}
+> 42.3 {{This always matches}
+> *    {{This never matches}}
+> ```
+
+As a user, I want to use a selector name that makes sense and feels natural to write.
+
+## Requirements
+
+_What properties does the solution have to manifest to enable the use-cases above?_
+
+## Constraints
+
+_What prior decisions and existing conditions limit the possible design?_
+
+ICU MF1 has a selector called `com.ibm.icu.text.SelectFormat` 
+associated with the keyword `select` in the MF1 syntax. 
+Our design should be compatible with this selector when used for the same use case,
+but it not constrained to use the same name or terminology.
+
+## Proposed Design
+
+_Describe the proposed solution. Consider syntax, formatting, errors, registry, tooling, interchange._
+
+The exact match selector is named `:string`.
```

Comment:
For selection, I think this is confusing. The alternative below of `exact` seems much clearer. I could also live with `equals`.